apps/root/src/pages/earn/components/strategies-table/components/toolbar/index.tsx

```diff
@@ -50,11 +49,8 @@ const AllStrategiesTableToolbar = ({
           </Typography>
         </ContainerBox>
       )}
-      <ContainerBox gap={6} alignItems="center" flexWrap="wrap" style={{ marginLeft: 'auto' }}>
-        <ContainerBox gap={3} alignItems="center">
-          <ReadyDelayedWithdrawals />
-          <PendingDelayedWithdrawals />
-        </ContainerBox>
+      <ContainerBox gap={6} alignItems="center" justifyContent="space-between" flexWrap="wrap">
```

Comment:
I think this space between is not making any effect, this is what i'm seing for 1440px viewport
<img width="1034" alt="Screenshot 2024-10-21 at 1 42 44 PM" src="https://github.com/user-attachments/assets/6be6b3e9-398f-411d-8e18-9f28f545c0f2">